app/src/main/res/layout/fragment_about_us.xml

```diff
@@ -2,75 +2,86 @@
   ~ This project is licensed under the open source MPL V2.
   ~ See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
   -->
-<androidx.coordinatorlayout.widget.CoordinatorLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:background="@drawable/bg_about"
-    android:layout_gravity="center"
-    android:layout_height="match_parent"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center"
+    android:background="@drawable/back"
```

Comment:
``` suggestion
```
@Prashant830 gradle build failed due to some error with drawable back. It might have not found during build. Please check this errors before commit with .\gradlew check --scan in terminal, also android studio shows error if project build fails. Please take time to remove errors, work with no hurry. Also look after build in pr. Don't do like make pr and forget. Thank you